polynote-kernel/src/main/scala/polynote/kernel/lang/python/PythonInterpreter.scala

```diff
@@ -183,7 +183,9 @@ class PythonInterpreter(val symbolTable: RuntimeSymbolTable) extends LanguageInt
         } else code
 
         jep.set("__polynote_code__", maybeModifiedCode)
-        jep.eval("exec(__polynote_code__, None, __polynote_locals__)\n")
+        kernelContext.runInterruptible {
+          jep.eval("exec(__polynote_code__, None, __polynote_locals__)\n")
+        }
```

Comment:
Hmm, I wonder whether it might be safer to just stick the `runInterruptible` inside `withJep`? That way we can interrupt anything that might happen when reading/writing values (just in case things hang before we exec somehow)

Reply:
Maybe once the `symbolTable.subscribe` is gone that would be fine, but for now I don't want PythonInterpreter taking over the kernel context task all the time

Reply:
Ohh I see, we can only run one interruptible thing at a time (for each KernelContext). Makes sense.